en/docs/learn/api-security/oauth2/access-token-types/jwt-tokens.md

```diff
@@ -24,6 +24,11 @@ The following prerequisites have to be satisfied for JWT based tokens to work.
       <p>The latter mentioned prerequisite is not applicable if you use the default certificates, which are the certificates that are shipped with the product itself. </p>
       </div> 
      </html>
+-  Enable the following configuration to the <NEW_API-M_HOME>/repository/conf/deployment.toml file of new WSO2 API Manager.
+
+    [apim.jwt]
```

Comment:
@arunpatil1983 , code should be added within code blocks. Therefore, this should be corrected as follows:

```
[apim.jwt]
enable = true
```

Reply:
@Mariangela -If code added within code block then it appears to the same line it do not go to the next line.
So that is reason the space is there between two lines.